text_extensions_for_pandas/io/test_bert.py

```diff
@@ -32,6 +34,7 @@
 )
 
 
+#@pytest.mark.skipif(sys.version_info[:2] <= (3, 6), reason="BertModel segfault")
```

Comment:
I'm getting a segfault locally when constructing the `BertModel`, maybe an env thing so see what happens in travis

Reply:
yup, looks like it's just my env :confused: